pkg/apis/postgres-operator.crunchydata.com/v1beta1/pgadmin_types.go

```diff
@@ -96,3 +96,10 @@ func (s *PGAdminPodSpec) Default() {
 		*s.Replicas = 1
 	}
 }
+
+// PGAdminPodStatus represents the observed state of a pgAdmin deployment.
+type PGAdminPodStatus struct {
+
+	// Identifies the users that have been installed into pgAdmin.
```

Comment:
:wrench: This wording threw me off a bit at first, since its not really possible to directly identify any specific users using the value/hash stored in this field.

Can we clarify that its a hash used to track the users that have been installed into pgAdmin?

Reply:
Makes sense; identity/identify. [StatefulSetStatus](https://pkg.go.dev/k8s.io/api/apps/v1#StatefulSetStatus) uses "indicates the version" on similar fields...

How about the following?

```suggestion
	// Hash indicating which users have been installed into pgAdmin.
```

Reply:
Works for me!